packages/neuron-wallet/src/store/WalletStore.ts

```diff
@@ -1,4 +1,5 @@
 import BaseStore from './store'
+import { KeyStore } from '../keys/keystore'
```

Comment:
`Keystore` is a de fact name of this concept, not `KeyStore`.

Reply:
OK